### PR TITLE
Docs: Use headings

### PR DESCRIPTION
**Pull request by jens-br**
This PR is a suggestion and uses headings instead of bold text in order to optimize the usability (user can reference single sections by himself easily).

This PR is based on #329 .

### CODE REVIEW

File: docs/deployer/deploy-docker-compose.rst [hunk 1]
```diff
@@ -19,14 +19,16 @@ Docker Compose needs the configuration file (``docker-compose.yml``).
...
   * `Docker Compose <https://docs.docker.com/compose/>`__
 
 |
 
 .. _preparation:
 
-**Preparation**
+Preparation
+-----------
+Prepare your local environment by installing Docker (if you have not installed it yet).
```
Comment (babsey) [inline docs/deployer/deploy-docker-compose.rst:31]: Add empty line

File: docs/deployer/deploy-docker-compose.rst [hunk 2]
```diff
@@ -36,7 +38,10 @@ Requirements
 
 .. _get-configuration-file:
 
-**Get the configuration file `docker-compose.yml` from GitHub**
+Get the configuration file 
+--------------------------
+The configuration file  `docker-compose.yml` contains all setup steps executed by Docker.
```
Comment (babsey) [inline docs/deployer/deploy-docker-compose.rst:43]: Add empty line.

File: docs/user/nest-models.rst [hunk 3]
```diff
@@ -29,7 +30,10 @@ Selected filter tags can be removed (click on close icon :guilabel:`x`).
 
 |br|
 
-**Filter tags for displaying models in the list**
+Setting filter tags in the model list
+-------------------------------------
+It is possible to select filter tags to display only models with certain properties.
```
Comment (babsey) [inline docs/user/nest-models.rst:35]: Add empty line.

**Review by babsey — CHANGES_REQUESTED** [review]
LGTM except small suggestions to add empty line after header

File: docs/deployer/deploy-docker-compose.rst (hunk 1 above)
Comment (jens-br) [reply docs/deployer/deploy-docker-compose.rst:32]: Please see [9e88a45](https://github.com/nest-desktop/nest-desktop/pull/330/commits/9e88a459b953571b9dec3c366abf3f0e44757891)

File: docs/deployer/deploy-docker-compose.rst (hunk 2 above)
Comment (jens-br) [reply docs/deployer/deploy-docker-compose.rst:45]: Please see [9e88a45](https://github.com/nest-desktop/nest-desktop/pull/330/commits/9e88a459b953571b9dec3c366abf3f0e44757891)

File: docs/user/nest-models.rst (hunk 3 above)
Comment (jens-br) [reply docs/user/nest-models.rst:36]: Please see [9e88a45](https://github.com/nest-desktop/nest-desktop/pull/330/commits/9e88a459b953571b9dec3c366abf3f0e44757891)

**Review by babsey — APPROVED** [review]
LGTM!

**Pull request merged**